extensions/wikia/CommunityPage/styles/entrypoint/EntryPoint.scss

```diff
@@ -1,61 +1,57 @@
 @import 'skins/shared/color';
 @import 'skins/oasis/css/core/breakpoints-variables';
 @import '../components/variables';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/base/typography';
 
-$background-color-dark: #1a1a1a;
-$background-color-light: #f6f6f6;
-
-$background-color: $background-color-light;
-@if $is-dark-wiki {
-	$background-color: $background-color-dark;
-}
-
-$logo-url-dark: '/extensions/wikia/CommunityPage/images/wikia-logo-dark.svg';
-$logo-url-light: '/extensions/wikia/CommunityPage/images/wikia-logo.svg';
-
-$logo-url: $logo-url-light;
-@if $is-dark-wiki {
-	$logo-url: $logo-url-dark;
-}
+$logo-url: '/extensions/wikia/DesignSystem/bower_components/design-system/dist/svg/wds-company-logo-fandom-heart.svg';
+$content-spacing-xsmall: 14px;
+$content-spacing-small: 20px;
+$content-spacing-regular: 30px;
+$content-spacing-large: 40px;
```

Comment:
Please, do not override existing variables, it might cause errors on auto refactoring and is misleading in general. Use new ones where needed.